src/OpenFeature/Model/TrackingEventDetails.cs

```diff
@@ -0,0 +1,129 @@
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+
+namespace OpenFeature.Model;
+
+/// <summary>
+/// The `tracking event details` structure defines optional data pertinent to a particular `tracking event`.
+/// </summary>
+/// <seealso href="https://github.com/open-feature/spec/blob/main/specification/sections/06-tracking.md#62-tracking-event-details"/>
+public sealed class TrackingEventDetails
+{
+    /// <summary>
+    /// The index for the "targeting key" property when the EvaluationContext is serialized or expressed as a dictionary.
+    /// </summary>
+    internal const string TargetingKeyIndex = "targetingKey";
```

Comment:
Do we need the `TargetingKeyIndex` here?

Reply:
No, we would need instead a `value` index if anything, that would serve the same purpose when the TrackingEventDetails are serialized.